math/FunMath.cpp

```diff
@@ -1 +1,11 @@
 #include "FunMath.h"
+
+
+int fibonacci(int n) {
+
+	if (n <= 1) {
+		return n;
+	} else {
+		return fibonacci(n-1) + fibonacci(n-2);
+
```

Comment:
If you are going to open this branch with a "{", you should also close it with a "}"